operator/config/manager/kustomization.yaml

```diff
@@ -1,14 +1,14 @@
-resources:
-- manager.yaml
-generatorOptions:
-  disableNameSuffixHash: true
-configMapGenerator:
-- files:
-  - controller_manager_config.yaml
-  name: manager-config
-apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
-images:
-- name: controller
-  newName: thundernetes-operator

```

Comment:
oops! was this generated somehow by previous commits?

Reply:
I guess so :(. I just ran the make create-install-files to ensure it was the most recent updates.